config.py

```diff
@@ -0,0 +1,31 @@
+
+import os
+
+from dotenv import load_dotenv
+
+TRELLO_URL = "https://api.trello.com/1/boards/{id}/cards"
+LIST_ID_COUNTER = 0
+CARD_ID_COUNTER = 0
+
+def get_trello_url():
+    return TRELLO_URL
+
+def get_trello_api_key():
+    return os.getenv("API_KEY")
+
+def get_trello_board_id():
+    return os.getenv("BOARD_ID")
+
+def get_trello_list_id():
+    return os.getenv("LIST_ID")
```

Comment:
You create new items using this and also mark items as done using this. So all items start done?